### PR TITLE
Add compat data for EventType

### PR DESCRIPTION
**Pull request by hichxm**
https://developer.mozilla.org/en-US/docs/Web/Events/click

### CODE REVIEW

File: api/EventType.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{
+  "api": {
+    "EventType": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Events",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
```
Comment (ExE-Boss) [inline api/EventType.json]: Can you put `webview_android` to the end of the support block? (see issue #398 and #1882)

File: api/EventType.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "click": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Events/click",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
```
Comment (ExE-Boss) [inline api/EventType.json]: ditto

**Comment by hichxm** [comment]
I have changed to alphanumeric order

File: api/EventType.json [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "click": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Events/click",
+        "support": {
```
Comment (ExE-Boss) [inline api/EventType.json]: The contents of `click` must be also contained inside a `__compat` block, that’s why [the build is failing](https://travis-ci.org/mdn/browser-compat-data/builds/370212795#L1098).

File: api/EventType.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "click": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Events/click",
```
Comment (ExE-Boss) [inline api/EventType.json]: The URL should be <code>https://developer.mozilla.org/docs/Web/Events/click</code>

File: api/EventType.json [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
```
Comment (ExE-Boss) [inline api/EventType.json]: `click` should have a sub-feature called `on_disabled_elements` containing the following:
```json
"on_disabled_elements": {
  "__compat": {
    "description": "On disabled form elements",
    "support": {
      "chrome": {
        "version_added": true,
        "partial_implementation": true,
        "notes": "Only works for <code>&lt;textarea&gt;</code> elements and some <code>&lt;input&gt;</code> element types."
      },
      "chrome_android": {
        "version_added": null
      },
      "edge": {
        "version_added": false
      },
      "edge_mobile": {
        "version_added": false
      },
      "firefox": {
        "version_added": false
      },
      "firefox_android": {
        "version_added": false
      },
      "ie": {
        "version_added": true,
        "partial_implementation": true,
        "notes": "Internet Explorer only triggers the <code>click</code> event on <code>&lt;input&gt;</code> elements of type <code>checkbox</code> or <code>radio</code> when the element is double-clicked."
      },
      "opera": {
        "version_added": true,
        "partial_implementation": true,
        "notes": "Only works for <code>&lt;textarea&gt;</code> elements and some <code>&lt;input&gt;</code> element types."
      },
      "opera_android": {
        "version_added": null
      },
      "safari": {
        "version_added": null
      },
      "safari_ios": {
        "version_added": null
      },
      "samsunginternet_android": {
        "version_added": null
      },
      "webview_android": {
        "version_added": null
      }
    },
    "status": {
      "experimental": false,
      "standard_track": true,
      "deprecated": false
    }
  }
}
```

File: api/EventType.json (hunk 4 above)
Comment (hichxm) [reply api/EventType.json]: fixed.

File: api/EventType.json (hunk 5 above)
Comment (hichxm) [reply api/EventType.json]: Why will I add this to the `click` event, is not sub-feature of `click`?

Comment (ExE-Boss) [reply api/EventType.json]: It must be on the same level as the `__compat` block, but not inside the `__compat` block.

Also, [stuff like this](https://developer.mozilla.org/en-US/docs/Web/Events/click#Browser_compatibility) has been made into sub-features in the past: https://github.com/mdn/browser-compat-data/blob/5133d14b23996400d17fead58193e7c27c8d0f22/api/NavigatorConcurrentHardware.json#L52-L56 https://github.com/mdn/browser-compat-data/blob/0fd82d0c7e966a44ce89dd14f9e33fb00375041f/html/elements/html.json#L110-L114

**Comment by Elchi3** [comment]
Thanks for your work! I thought this was about an "EventType" API. It isn't however. 

We have not yet decided how to structure compat data from MDN event pages. See the discussion in https://github.com/mdn/browser-compat-data/issues/935. I will come back to this PR once that is sorted. Sorry for any inconveniences.

**Comment by hichxm** [comment]
No problem, take your time. the pull request does not move.

**Comment by Elchi3** [comment]
We've started re-doing event pages and mouse events are reworked as part of this task: https://github.com/mdn/sprints/issues/955 
The compat data will be submitted to this repo as part of that work as well. The page is already moved to a new place https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event

I'm closing here now as we won't add this data the way this PR suggests.

**Pull request closed without merging**